app/javascript/components/TopicSectionSuggestion/index.jsx

```diff
@@ -1,58 +1,58 @@
-import React, {useState} from "react"
+import React, {useState, useEffect} from "react"
 import TopicPageContent from "../TopicPageContent"
 import Button from '../Button'
 import "./style.scss"
 
 const useStateWithLocalStorage = localStorageKey => {
-  const [resultsStored, setResultsStored] = React.useState(
+  const [resultsStored, setResultsStored] = useState(
     localStorage.getItem(localStorageKey) || ''
   );
 
-  React.useEffect(() => {
+  useEffect(() => {
     localStorage.setItem(localStorageKey, resultsStored);
   }, [resultsStored]);
 
   return [resultsStored, setResultsStored];
 };
 
-const TopicSectionSuggestion = ({
-    healthtitle,
-    healthintro,
-    familytitle,
-    familyintro,
-    hometitle,
-    homeintro,
-    timetitle,
-    timeintro,
-    skillstitle,
-    skillsintro,
-    currentSection,
-    allComplete }) => {
+const TopicSectionSuggestion = ({ currentSection }) => {
   const family = "results_topic_family";
   const home = "results_topic_home";
   const health = "results_topic_lifestyle";
   const time = "results_topic_time";
   const skills = "results_topic_skills";
+  const children = "results_topic_children";
   const [familyResultsStored, setFamilyResultsStored] = useStateWithLocalStorage(family);
   const [homeResultsStored, setHomeResultsStored] = useStateWithLocalStorage(home);
   const [healthResultsStored, setHealthResultsStored] = useStateWithLocalStorage(health);
   const [timeResultsStored, setTimeResultsStored] = useStateWithLocalStorage(time);
   const [skillsResultsStored, setSkillsResultsStored] = useStateWithLocalStorage(skills);
-  var amountComplete = 0;
-  if(healthResultsStored.length) { amountComplete++; }
-  if(timeResultsStored.length) { amountComplete++; }
-  if(familyResultsStored.length) { amountComplete++; }
-  if(skillsResultsStored.length) { amountComplete++; }
-  if(homeResultsStored.length) { amountComplete++; }
+  const [childrenResultsStored, setChildrenResultsStored] = useStateWithLocalStorage(children);
+
+  let amountComplete = 0;
+
+  const arr = [healthResultsStored, timeResultsStored, familyResultsStored, skillsResultsStored, childrenResultsStored, homeResultsStored]
+
+  for (let i = 0; i < arr.length; i++) {
+    if (i.length) {
```

Comment:
I think this needs to be:

```suggestion
    if (arr[i].length) {
```

Reply:
Done